Gemfile.lock

```diff
@@ -54,7 +54,7 @@ GEM
     jekyll-mentions (1.6.0)
       html-pipeline (~> 2.3)
       jekyll (>= 3.7, < 5.0)
-    jekyll-multiple-languages-plugin (1.8.0)
+    jekyll-multiple-languages-plugin (1.7.0)
```

Comment:
The lockfile is generated automatically and not really intended to be edited manually. I would rather configure this from `Gemfile` like this:

```
gem "jekyll-multiple-languages-plugin", "1.7.0"
```

Then it ensures that the version stays the same even when updating components with `bundler update`